benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -892,8 +894,14 @@ void GraphicsBenchmarkApp::ProcessKnobs()
         pDrawCallCount->SetValue(pSphereInstanceCount->GetValue());
     }
 
+    // If the debug view is enabled, then change the pixel shader to PsSimple.
```

Comment:
Shall the knob be disabled be this is on?
(Instead of ignoring/reseting its value?)

Reply:
That is true. Updated.